src/vue/modules/issuance-explorer/components/issuances-table.vue

```diff
@@ -80,3 +90,7 @@ export default {
   },
 }
 </script>
+
+<style lang="scss">
```

Comment:
It's not necessary to add empty styles to a component. It requires only template & script parts